.github/workflows/lint.yaml

```diff
@@ -0,0 +1,34 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    name: Lint the app
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Ruby 
+      uses: ruby/setup-ruby@v1 # no need for ruby-version because we have a .ruby_version file
+      with:
+          bundler-cache: true
+
+    - name: Cache dependencies
+      uses: actions/cache@v3
+      with: 
+        path: vendor/bundle
+        key: ${{ runner.os }}-bundle-{{ hashFiles("**/Gemfile.lock") }}
+        restore-keys: |
+          ${{ runner.os }}-${{ hashFiles('**/Gemfile.lock') }}
+          ${{ runner.os }}-${{ github.ref_name }}
+          ${{ runner.os }}-main
+          ${{ runner.os }}-
+  
+    - name: Run Rubocop
+      run: bundle exec rubocop --config ./.rubocop.yml
```

Comment:
LGTM only thing is the no new line at the end of the file but that might just be a yaml linting thing